src/tests/version.rs

```diff
@@ -113,18 +114,18 @@ fn record_rerendered_readme_time() {
 
 #[test]
 fn version_size() {
-    let mut session = MockUserSession::logged_in();
+    let (_, _, user) = TestApp::with_user();
```

Comment:
I'm not a huge fan of these places that we have to do `let (_, _, ...`, but I don't see enough of a pattern yet to have a suggestion for how to get rid of them :-/

Reply:
Yeah, I see pros and cons here.  I do like that it is easy to look at the first line of a test and know pretty quickly that the test does not access the database directly (app is dropped), and makes only authenticated API request (anon is dropped).  It certainly does look strange in a few cases like this though.

I considered adding some sort of `SimpleTest` constructor that could hide some of this, but I'm not really sure what that should look like yet or if it is really necessary.  Hopefully as more tests are converted we will get a better feel for exactly what would be useful.